samtranslator/model/api/api_generator.py

```diff
@@ -98,9 +97,9 @@ def _construct_rest_api(self):
                                            "Specify either 'DefinitionUri' or 'DefinitionBody' property and not both")
 
         if self.open_api_version:
-            if re.match(OpenApiVersionsSupportedRegex, self.open_api_version) is None:
+            if re.match(SwaggerEditor.get_openapi_versions_supported_regex(), self.open_api_version) is None:
                 raise InvalidResourceException(
-                    self.logical_id, "The OpenApiVersion value must be of the format 2.x.x or 3.x.x")
+                    self.logical_id, "The OpenApiVersion value must be 3.0.0")
```

Comment:
"must be of the format 3.0.0" is fine; it's just that "2.x.x" was ambiguous.

Reply:
done. thanks!